base/stacktraces.jl

```diff
@@ -96,13 +96,13 @@ end
 
 
 """
-    lookupat(pointer::Union{Ptr{Cvoid}, UInt}) -> Vector{StackFrame}
```

Comment:
I think we'll need to keep this around with a soft deprecation then hard deprecate it in 2.0.

Reply:
This is reverting it back to the original name. There hasn't been a release with this.

Reply:
Ah, okay, great.

Reply:
Yes; for more context see https://github.com/JuliaLang/julia/pull/33190#discussion_r322007723 and https://github.com/JuliaLang/julia/pull/33190#discussion_r322299303